src/main/java/org/dependencytrack/model/Component.java

```diff
@@ -128,44 +127,39 @@ public enum FetchGroup {
     private OrganizationalEntity supplier;
 
     @Persistent
-    @Column(name = "GROUP", jdbcType = "VARCHAR")
+    @Column(name = "GROUP", jdbcType = "CLOB")
     @Index(name = "COMPONENT_GROUP_IDX")
-    @Size(max = 255)
     @Pattern(regexp = RegexSequence.Definition.PRINTABLE_CHARS, message = "The group may only contain printable characters")
     private String group;
 
     @Persistent
-    @Column(name = "NAME", jdbcType = "VARCHAR", allowsNull = "false")
+    @Column(name = "NAME", allowsNull = "false", jdbcType = "CLOB")
     @Index(name = "COMPONENT_NAME_IDX")
     @NotBlank
-    @Size(min = 1, max = 255)
     @JsonDeserialize(using = TrimmedStringDeserializer.class)
     @Pattern(regexp = RegexSequence.Definition.PRINTABLE_CHARS, message = "The name may only contain printable characters")
     private String name;
 
     @Persistent
-    @Column(name = "VERSION", jdbcType = "VARCHAR")
-    @Size(max = 255)
+    @Column(name = "VERSION", jdbcType = "CLOB")
     @JsonDeserialize(using = TrimmedStringDeserializer.class)
     @Pattern(regexp = RegexSequence.Definition.PRINTABLE_CHARS, message = "The version may only contain printable characters")
     private String version;
 
     @Persistent
-    @Column(name = "CLASSIFIER", jdbcType = "VARCHAR")
+    @Column(name = "CLASSIFIER")
```

Comment:
Should this also be `CLOB`?

Reply:
If not specified as well, it works fine. But to keep consistency I agree that we should specify CLOB.